sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -557,6 +562,7 @@ class ParquetFilters(
         } else {
           Some(sources.Or(leftResultOptional.get, rightResultOptional.get))
         }
+
```

Comment:
unnecessary change.

Reply:
Just break a new line. Only for better code reading.

Reply:
removed now